consensus/hotstuff/cruisectl/config.go

```diff
@@ -0,0 +1,57 @@
+package cruisectl
+
+import (
+	"math"
+	"time"
+)
+
+// DefaultConfig returns the default config for the BlockRateController.
+func DefaultConfig() *Config {
+	return &Config{
+		TargetTransition: EpochTransitionTime{
+			day:    time.Wednesday,
+			hour:   19,
+			minute: 0,
+		},
+		// TODO confirm default values
+		DefaultBlockRateDelay: 500 * time.Millisecond,
+		MaxDelay:              1000 * time.Millisecond,
+		MinDelay:              250 * time.Millisecond,
+		Enabled:               true,
+		N:                     600, // 10 minutes @ 1 view/second
+		KP:                    math.NaN(),
+		KI:                    math.NaN(),
+		KD:                    math.NaN(),
+	}
+}
+
+// Config defines configuration for the BlockRateController.
+type Config struct {
+	// TargetTransition defines the target time to transition epochs each week.
+	TargetTransition EpochTransitionTime
+	// DefaultBlockRateDelay is the baseline block rate delay. It is used:
+	//  - when Enabled is false
+	//  - when epoch fallback has been triggered
+	//  - as the initial block rate delay value, to which the compensation computed
+	//    by the PID controller is added
+	DefaultBlockRateDelay time.Duration
```

Comment:
What do you think about `DefaultProposalDelay`? Concretely, it is the delay introduced while proposing. It only _contributes_ to a change in the block rate / block time, which is a network-wide measurement. 

I don't think the units here should be in `views/second` because we can only directly influence the delay in the time of an individual view. Since we want to have the manual fallback (disable the controller completely), and that manual fallback is only applying a delay on a view-by-view basis, using the proposal time / block time units is more intuitive to me.

Reply:
Updated the terminology here: https://github.com/onflow/flow-go/pull/4332/commits/173694e8e5b2081b329fbfc54478e1958eecd182